### PR TITLE
Assign attributes directly during import validation

### PR DESCRIPTION
**Pull request by trandoanhung1991**
To prevent parsing UTC time into local datetime

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -1,76 +1 @@
...
-
-* Blythe Dunham
-* Gabe da Silveira
-* Henry Work
-* James Herdman
-* Marcus Crafter
-* Thibaud Guillaume-Gentil
-* Mark Van Holstyn 
-* Victor Costan
+Tioeuaeuesting webhook to Jenkins
```
Comment (jkowens) [inline README.markdown]: I don't think you meant to commit this?

**Comment by jkowens** [comment]
@trandoanhung1991 If you remove your change to README.markdown and squash this into a single commit that would probably help move this along. I think there's been another issue discovered that this fix could resolve. See #235. 

I would also suggest referencing the related issue #140 that you opened in the description. :smile:

**Comment by trandoanhung1991** [comment]
For some reason I can't find this repo anywhere. I'll close this PR and create a new one, is that ok?

**Comment by trandoanhung1991** [comment]
Just in case, the new PR is #237

**Pull request closed without merging**